src/components/tooltip/tooltip.spec.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { render, fireEvent } from '../../test-utils';
+import Tooltip from './tooltip';
+
+jest.useFakeTimers();
+
+// copy pasta from https://github.com/FezVrasta/popper.js/issues/478
+jest.mock(
+  'popper.js',
+  () =>
+    class Popper {
+      static placements = [
+        'auto',
+        'auto-end',
+        'auto-start',
+        'bottom',
+        'bottom-end',
+        'bottom-start',
+        'left',
+        'left-end',
+        'left-start',
+        'right',
+        'right-end',
+        'right-start',
+        'top',
+        'top-end',
+        'top-start',
+      ];
+
+      constructor() {
+        return {
+          destroy: () => {},
+          scheduleUpdate: () => {},
+        };
+      }
+    }
+);
```

Comment:
Maybe put into a `ui-kit/mocks/applyPopperMocks.js` cause then it could be reused.

Reply:
Yeah good call 👍